public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -394,30 +407,18 @@ export class DashboardModel {
   exitViewPanel(panel: PanelModel) {
     this.panelInView = undefined;
     panel.setIsViewing(false);
-    this.refreshIfChangeAffectsAllPanels();
   }
 
   exitPanelEditor() {
     this.panelInEdit!.destroy();
     this.panelInEdit = undefined;
-    this.refreshIfChangeAffectsAllPanels();
     getTimeSrv().resumeAutoRefresh();
   }
 
-  setChangeAffectsAllPanels() {
-    if (this.panelInEdit || this.panelInView) {
-      this.hasChangesThatAffectsAllPanels = true;
-    }
-  }
-
-  private refreshIfChangeAffectsAllPanels() {
-    if (!this.hasChangesThatAffectsAllPanels) {
-      return;
-    }
-
-    this.hasChangesThatAffectsAllPanels = false;
-    this.startRefresh();
-  }
+  /*
+   * @deprecated used to be used internally by variables
+   * */
+  setChangeAffectsAllPanels() {}
```

Comment:
I think it's fine to remove this, can't imagine any plugin uses it

Reply:
Ok, I'll do that.